ramls/eholdings.raml

```diff
@@ -923,6 +1095,18 @@ traits:
           body:
             application/vnd.api+json:
               description: The server has successfully fulfilled the PUT request.
+              example: |
```

Comment:
I am confused by this response for a few reasons: 
1. A successful PUT request to `providers` gives a status code 200 if it is successful.
2. And to double check when I tried a PUT on `/eholdings/providers/18`, I got a success 200 OK with response below: 
``{"data":{"id":"18","type":"providers","attributes":{"name":"Gale | Cengage","packagesTotal":243,"packagesSelected":18,"providerToken":{"factName":"[[galesiteid]]","prompt":"/itweb/","helpText":"\u003cul\u003e\r\n    \u003cli\u003eEnter your Gale\u003csup\u003e®\u003c/sup\u003e site ID in the space provided below. The site ID may contain a combination of alpha/numeric characters, varying in length. \u003cblockquote style=\"margin-right: 0px;\" dir=\"ltr\"\u003e\r\n    \u003cp\u003e Example: The site ID immediately follows /itweb/ in a URL. The site ID in the following URL is \u003ci\u003eaa11bb22\u003c/i\u003e. \u003c/p\u003e\r\n    \u003c/blockquote\u003e\u003c/li\u003e\r\n\u003c/ul\u003e\r\n\u003cblockquote style=\"margin-right: 0px;\" dir=\"ltr\"\u003e\u003cblockquote style=\"margin-right: 0px;\" dir=\"ltr\"\u003e\r\n\u003cp\u003e\u003cspan style=\"text-decoration: underline;\"\u003ehttp://infotrac.galegroup.com/itweb/aa11bb22?db=AIM\u003c/span\u003e\u003c/p\u003e\r\n\u003c/blockquote\u003e\u003c/blockquote\u003e\u003cbr /\u003e\r\n\u003cul\u003e\r\n    \u003cli\u003eIf no site ID is specified, your Gale Group links may not function properly, as Gale Group requires this information for authentication. \u003c/li\u003e\r\n    \u003cli\u003eIf you are unable to locate the site ID, please contact Gale Group. For contact information, visit: \u003ca href=\"http://access.gale.com/authentication/\"\u003ehttp://access.gale.com/authentication/\u003c/a\u003e. \u003c/li\u003e\r\n\u003c/ul\u003e\r\n","value":"99"},"supportsCustomPackages":false,"proxy":{"id":"\u003cn\u003e","inherited":false}},"relationships":{"packages":{"meta":{"included":false}}}},"jsonapi":{"version":"1.0"}}``` which differs from the structure below.
3. The example you have below looks like request payload and not response.

Reply:
This was my mistake, thank you for catching it. I will update with the correct information.